### PR TITLE
diff-npm-package: create reports folder

### PR DESCRIPTION
**Pull request by yaacovCR**

### CODE REVIEW

**Review by n1ru4l — APPROVED** [review]

**Review by saihaj — APPROVED** [review]

File: resources/diff-npm-package.ts [hunk 1]
```diff
@@ -33,6 +33,9 @@ if (diff === '') {
   console.log('No changes found!');
 } else {
   const reportPath = localRepoPath('reports', 'npm-dist-diff.html');
+  if (!fs.existsSync) {
```
Comment (IvanGoncharov) [inline resources/diff-npm-package.ts:36]: @yaacovCR Did you test it?
Are you checking that `fs.existsSync` exist as a function or you checking if `reportPath` exists?

Also PR doesn't contain any description.

**Review by IvanGoncharov — CHANGES_REQUESTED** [review]

File: resources/diff-npm-package.ts (hunk 1 above)
Comment (yaacovCR) [reply resources/diff-npm-package.ts:36]: Whoops! Sorry about that. Feel free to take over and fix as I won’t be able to address until several hours from now.

Comment (saihaj) [inline resources/diff-npm-package.ts:36]:
```suggestion
  if (!fs.existsSync(reportPath)) {
```

**Review by saihaj — APPROVED** [review]

**Comment by yaacovCR** [comment]
this is still actually (sadly) not working, although CI now is.

https://github.com/graphql/graphql-js/pull/3623/checks#step:7:7
> Run actions/upload-artifact@v[3](https://github.com/graphql/graphql-js/pull/3623/checks#step:7:3)
  with:
    name: npm-dist-diff.html
    path: ./reports/npm-dist-diff.html
    if-no-files-found: ignore
No files were found with the provided path: ./reports/npm-dist-diff.html. No artifacts will be uploaded.

will try to submit a fix

**Pull request merged**